packages/cdk/lib/stacks/core-stack.ts

```diff
@@ -55,6 +55,13 @@ export interface CoreStackProps extends StackProps {
    * @default none
    */
   parameters?: ParameterProps[];
+  /**
+   * If true, use public subnets instead of private subnets, and do not create the VPC endpoints and the NAT gateway.
+   * This will result in significantly lower ongoing costs when no job is running.
```

Comment:
Same comment as above regarding why costs are lower and potential security warning